docs/dev/tools/common-errors.md

```diff
@@ -184,27 +184,13 @@ This can happen when there is a slow internet connection, when the module is try
 **Solution:**
 
 - Try using a faster internet connection.
-- If the problem persists, try increasing the `VIAM_MODULE_STARTUP_TIMEOUT` or `VIAM_RESOURCE_CONFIGURATION_TIMEOUT` environment variables in the module's JSON configuration file.
-  For example:
-
-  ```json {class="line-numbers linkable-line-numbers"}
-  {
-    "modules": [
-      {
-        ...
-        "env": {
-          "VIAM_MODULE_STARTUP_TIMEOUT": "6m30s",
-          "VIAM_RESOURCE_CONFIGURATION_TIMEOUT": "3m0s"
-        }
-      }
-    ]
-  }
-  ```
-
 - If you are the module author, consider packaging the module with required dependencies so they don't need to be downloaded on startup.
   For Python modules, you can package your module with dependencies by using the PyInstaller steps when [uploading your module](/operate/get-started/other-hardware/#upload-your-module).
+- If the problem persists, try setting the `VIAM_MODULE_STARTUP_TIMEOUT` or `VIAM_RESOURCE_CONFIGURATION_TIMEOUT` environment variables on your machine.
+  For a value, pass a sequence of numbers and units, e.g. "6m30s", "6.5m", or "630000ms".
+  You can specify units in "ns" (nanoseconds), "us"/"µs" (microseconds), "ms" (milliseconds), "s" (seconds), "m" (minutes), and "h" (hours). 
```

Comment:
Might actually be more useful with fewer words to show an example of the complete syntax you'd pass in the command line and/or in the service config file, and not worry about the different possible time units since those seem like general convention?

Reply:
Great point. I'll just show one example with m, s, and ms to keep things brief. lord help us if someone sets a timeout in ns 😆